api/v2/sdks/client-side-identity.md

```diff
@@ -95,10 +117,10 @@ Here's what you need to know about this function:
 - You can call `init()` any time after the SDK has been loaded by the corresponding script tag, typically during page loading.
 - Initialization calls require a [callback function](#callback-function) that is invoked after the SDK is initialized.
 - When creating an instance for the UID2 lifecycle on the client, the `identity` property in the `init()` call refers to the `body` property of the response JSON object returned from a successful [POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) call with the server-side generated identity.
-- Since the SDK relies on [first-party cookies](#uid2-cookie-format) to store the passed UID2 identity information for the session, subsequent `init()` calls on different page loads may have the `identity` property empty.
-- To tune specific behaviors, initialization calls may include optional configuration [parameters](#parameters).
+- Since the SDK relies on [first-party cookies](#uid2-cookie-format) to store the passed UID2 identity information for the session, subsequent `init()` calls on different page loads might have the `identity` property empty. {**GWH_03_AT: I don't totally understand this. Is it important? Anything users should do to help prevent it?**}
```

Comment:
> Since the SDK relies on [first-party cookies](#uid2-cookie-format) to store the passed UID2 identity information for the session, calls to `init()` made by pages on different domains may not be able to access the cookie. You can adjust the settings used for the cookie with the `cookieDomain` and `cookiePath` options.

Reply:
Update:
Since the SDK relies on [first-party cookies](#uid2-cookie-format) to store the passed UID2 information for the session, a call to `init()` made by a page on a different domain might not be able to access the cookie. You can adjust the settings used for the cookie with the `cookieDomain` and `cookiePath` options.